mne/io/base.py

```diff
@@ -1450,6 +1451,72 @@ def save(self, fname, picks=None, tmin=0, tmax=None, buffer_size_sec=None,
                    start, stop, buffer_size, projector, drop_small_buffer,
                    split_size, split_naming, 0, None, overwrite)
 
+    def export(self, fname, fmt="auto"):
+        """Export Raw to external formats.
+
+        Supported formats: EEGLAB (set)
+
+        .. warning::
+            Since we are exporting to external formats, there's no
+            guarantee that all the info will be preserved in the external
+            format. To save in native MNE format (fif) without info loss,
+            use :func:`save` instead.
+
+        Parameters
+        ----------
+        fname : str
+            Name of the export file.
+        fmt : str
+            Format of the export. Defaults to auto, which will infer
+            format from fname's extension.
+
+        Notes
+        -----
+        For EEGLAB format, channel locations are expanded to full EEGLAB
+        format. For more details see :func:`eeglabio.utils.cart_to_eeglab`.
```

Comment:
We have the same docstring in the method. Can you please move it into the `docdict` (in `mne/utils/docs.py`) to remove the redundancy?

Reply:
Since raw and epochs can technically have different export targets in the future, should I just move the warning and eeglab's notes to `docdict`?

Reply:
Also, I looked around but I'm not quite sure how to use them, are there any examples elsewhere in the code that I can copy from?

Reply:
I would move the entire thing for now, except for the very first line, which is different.

You add the documentation string to an entry in the `docdict`, and then you can reference it in the function's/method's docstring via `%(the_docdict_key_you_picked)s`

If the function/method has a `@verbose` or `@fill_doc` decorator, the `%(...)s` will be replaced with the corresponding bits from the `docdict` on documentation build time. 

I'd suggest to add the `@verbose` decorator and a `verbose` kwarg to allow users to control the verbosity level; and this then enables the `docdict` support automatically :)

Reply:
What does the verbose kwarg do exactly? Isn't this supposed to be used for logging?

Reply:
I decided to only move the parts that don't contain supported formats to `docdict` and used `@verbose` (still not entirely sure what it does). Everything else should work now.

Reply:
we have 2 decorators, `@verbose` and `@fill_doc`. You need one of them in order for docdict replacements to work.  The `@verbose` decorator additionally adds a `verbose` kwarg, which (as you thought) allows users to control logging.  `verbose` is usually passed down to private/helper function calls, so even if there are no `warn()` or `logger` calls in the top-level function, `verbose` can still affect how much output a user sees.